### PR TITLE
Combine unit and integration tests

### PR DESCRIPTION
**Pull request by terrytangyuan**
Most of these files are not covered in unit tests but covered in integration tests instead so we should not take them into account.

Note that somehow the uploaded coverage report is still outdated so I added some additional tests and the issue has been fixed.

Signed-off-by: terrytangyuan <terrytangyuan@gmail.com>

### CODE REVIEW

**Comment by terrytangyuan** [comment]
Somehow the uploaded coverage report is still outdated. I am adding some additional tests to see if the issue is fixed.

**Comment by ywskycn** [comment]
> Most of these files are not covered in unit tests but covered in integration tests instead so we should not take them into account.

You mean these files will bring down the coverage rate?

**Comment by terrytangyuan** [comment]
@ywskycn Yes, previously the coverage percentage was around 50% which is pretty low and it has been boosted to 80% after this PR. Most of those code rely on Minikube and ODPS and they are actually covered in Travis.

File: .codecov.yml [hunk 1]
```diff
@@ -13,5 +13,16 @@ coverage:
 
 ignore:
   - "elasticdl/python/tests"
+  - "elasticdl/python/common/k8s_client.py"
```
Comment (wangkuiyi) [inline .codecov.yml]: May we know why should we exclude these source files from unit test coverage stats?

**Comment by ywskycn** [comment]
> @ywskycn Yes, previously the coverage percentage was around 50% which is pretty low and it has been boosted to 80% after this PR. Most of those code rely on Minikube and ODPS and they are actually covered in Travis.

But that should be fine, right? Otherwise, we always need to think about whether we should exclude newly added files or not.

**Comment by terrytangyuan** [comment]
@ywskycn @wangkuiyi I think we can probably combine non-k8s and k8s-related tests so the relevant code doesn't have to be excluded in coverage calculation. For ODPS related code, we still have to exclude them since Travis builds will skip ODPS related tests for pull requests from forks. What do you think?

**Comment by terrytangyuan** [comment]
Discussed with @ywskycn offline, I combined all tests to one stage on Travis and the k8s related code should be included and covered in test coverage report now. No code will be excluded from test coverage anymore.

**Review by ywskycn — APPROVED** [review]

**Pull request merged**